chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertificationIdController.java

```diff
@@ -57,8 +57,9 @@
     //
     // Retrieves all CMS Certification IDs and their date of creation.
     // **********************************************************************************************************
-    @PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_ONC', 'ROLE_CMS_STAFF')")
-    @ApiOperation(value = "Retrieves a list of all CMS EHR Certification IDs along with the date they were created.")
+    @PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_ONC', 'ROLE_CMS_STAFF', 'ROLE_ONC_STAFF')")
```

Comment:
Didn't we get rid of ROLE_ONC_STAFF? I think the PreAuthorize was correct.

Reply:
Not sure what I did here!  Should be fixed, though.